scripts/secchiLagosUniqueNHD.R

```diff
@@ -3,8 +3,8 @@ library(rgdal)
 #get nhd layer
 nhd <- readOGR(dsn = paste0(getwd(),"/data"), layer="NHDWaterbody")
 
-#read in lagos data
-input <- read.delim("secchi_subset.tsv")
+#read in lagos data previously downloaded from sciencebase
+input <- read.delim("secchi_LAGOS_subset.tsv")
```

Comment:
would like to capture the whole thing here, including pointing to the item on sciencebase and using `sbtools` to grab it